plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/wizard/NewDataflowProjectWizard.java

```diff
@@ -59,13 +60,13 @@ public boolean performFinish() {
     }
     try {
       getContainer().run(true, true, creator);
+      return true;
     } catch (InvocationTargetException | InterruptedException ex) {
-      // TODO: handle
-      DataflowUiPlugin.logError(ex, 
-          "Error encountered when trying to create project"); //$NON-NLS-1$
+      String message = "Error encountered when trying to create project"; //$NON-NLS-1$
```

Comment:
I think we should swap the order here: `StatusUtil#setErrorStatus()` logs the error too, but it will be unwrapped.  That way we'll see the unwrapped exception first, then the logged status.